spec/searchers/responses_searcher_spec.rb

```diff
@@ -174,4 +254,12 @@ def search(query)
   def searcher(query)
     ResponsesSearcher.new(relation: Response, query: query, scope: {mission: get_mission})
   end
+
+  def items_by_name_and_id(*items)
+    items.pluck(:id, :name).map { |id, name| {id: id, name: name} }
+  end
+
+  def qing_for_question(question)
+    FilterDataController.filter_unique(Questioning.where(question: question)).first
+  end
```

Comment:
I don't think this should be necessary unless I'm missing something.

Reply:
You're right -- I added it to use the deterministic quality of `filter_unique`, but during these tests there's only one questioning per question so the determinism isn't needed. I'll remove.